File: ProjekatAgencija/src/app/entities/aircompany/aircompany.ts

export class AirCompany {
    Id: number;
    Name: string;
    Address: string;
    About: string;
    Prices: string;
    Destinations: Array<Destination>;
    Flights: Array<Flight>;
    Ratings: Array<Rating>;  

    public constructor(init?: Partial<AirCompany>) {
        Object.assign(this, init);
    }
}

export class Destination{
    Id: number;
    Destination1: string;

    //AirCompany: AirCompany;

    public constructor(init?: Partial<Destination>) {
        Object.assign(this, init);
    }
}

export class Flight{
    Id: number;
    TakeoffDate: Date;
    LandingDate: Date;
    TakeoffTime: Date;
    FlightLength: number;
    TicketPrice: number;
    Ratings: Array<Rating>;  

    public constructor(init?: Partial<Flight>) {
        Object.assign(this, init);
    }
}

export class Rating{
    Id: number;
    Rating1: number;

    public constructor(init?: Partial<Rating>) {
        Object.assign(this, init);
    }
}

File: DBProjekat/DBProjekat/Models/AirCompany.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace DBProjekat.Models
{
    public class AirCompany
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Address {get; set; }
        [Required]
        public string About { get; set; }
        [Required]
        public string Prices { get; set;  }
        public List<Destination> Destinations { get; set; }
        public List<Flight> Flights { get; set; }
        public List<Rating> Ratings { get; set; }
    }
}
